### ISSUE TITLE
It is not possible to drag on the element that will exists only after the drag starts

### ISSUE THREAD

**Issue by xMort** [DevExpress/testcafe#2225]
### Are you requesting a feature or reporting a bug?
Feature

### What is the current behavior?
There are two dragging functions available on the test controller: drag and dragToElement.

### What is the expected behavior?
When I try to drag to an element that does not exist yet the dragging fails with "The specified "destinationSelector" does not match any element in the DOM tree." error. 

The issue is that my target element will be appended to the DOM tree only after the dragging starts. Is there a way how can I successfully test this behavior in TestCafe? I cannot use simple drag with XY offset because destination element position differs based on the browser window size which is different in CI and development environment.

### Specify your environment

* operating system: MacOS High Sierra, latest Google Chrome browser
* testcafe version: 0.18.5
* node.js version: 8.9.4

**Comment by AndreyBelym** [comment]
Hi @xMort! This issue looks like a duplicate of https://github.com/DevExpress/testcafe/issues/1374, so I will close your issue. Feel free to reopen it if #1374 doesn't meet your needs.
